Source/VMware.vSphereDSC/Tests/Integration/vCenterSettings.Integration.Tests.ps1

```diff
@@ -1,9 +1,9 @@
 <#
-Copyright (c) 2018 VMware, Inc.  All rights reserved				
+Copyright (c) 2018 VMware, Inc.  All rights reserved
```

Comment:
If you can just fix the indentation based on the Formatting_Guidelines, so it it not shown in the diff.